pages/models/generations/Sources/VSC/PhasorGridFollowingVSC/PhasorIo/index.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: page 
+title: Phasor-I0 Grid Following Voltage Source Converter 
+tags: [Opensource, Phasor, voltage source, converter, wind, pv, hdvc, dynawo, STEPSS] 
+date: 26/06/2024 
+last-updated: 01/07/2024
+id: #165
+authors: Carlos Alegre (eRoots)
+reviewers: Eduardo Prieto Araujo (UPC), Josep Fanals Batllori (eRoots)
+---
+
+
+## Context
+
+The Phasor-I0 Grid Following Voltage Source Converter (VSC) model is derived from the Full-Phasor described [here](../). 
+
+## Model use, assumptions, validity domain and limitations
+
+The model has uses similar to the $$I_1$$ model, with some improvement in the simulation speed while the precision is slightly reduced.
+
+The assumptions made starting from the Full-Phasor model are the following:
+
+* The current loop is completely removed from the model. The current obtained from the outer loop is directly considered to be provided by the VSC.
+* Since there are no current dynamics, the outer loop is modified to mantain the closed-loop from the Full-Phasor model.
+
+The model is valid for low-frequency phenomena studies. The high-frequency events are not captured as well as in the Full-Phasor model for instance, as shown in the several Figures of [[1]](#1). 
+
+## Model description
+
+The diagram of the complete model is shown in the following figure.
+
+<div style="background-color:rgba(0, 0, 0, 0); text-align:center; vertical-align: middle; padding:4px 0;">
+<img src="{{ '/pages/models/generations/Sources/VSC/PhasorGridFollowingVSC/PhasorIo/PhasorI0.svg' | relative_url }}"
+     alt="Phasor-I0 Model Block Diagram"
+     style="float: center; margin-right: 10px; width: 600px;" />
+</div>
+<div align = 'center'>
+Figure 1: Phasor-I0 Model Block Diagram <a href="#1">[1]</a>
+</div>
+<br>
+
+The following subsections will describe the changes with respect to the Full-Phasor model.
+
+### Current loop
+
+The block is completely removed, which means $$i^{qd*} = i^{qd*}_c $$.
+
+### Outer loop
+
```

Comment:
do we have the same input /output variables here as from the complete scheme? if not, could you add an input output diagram?

Reply:
added reference to complete scheme